lua/starfall/libs_cl/render.lua

```diff
@@ -2116,10 +2245,8 @@ end
 -- @param number y Pixel y-coordinate.
 -- @return Color Color object with ( r, g, b, a ) from the specified pixel.
 function render_library.readPixel(x, y)
-	if not renderdata.isRendering then
-		SF.Throw("Not in rendering hook.", 2)
-	end
-	return cwrap(Color(render.ReadPixel(x, y)))
+	if not renderdata.isRendering then SF.Throw("Not in rendering hook.", 2) end
+	return setmetatable({render.ReadPixel(x, y)}, col_meta)
```

Comment:
Might be worth adding a 'readPixelRGBA' function that returns 4 rgba numbers, for more performance if needed.

Reply:
done